### ISSUE TITLE
SortableTable React Component should allow one-dimensional table data

### ISSUE THREAD

**Issue by d-reinhold** [pivotal-cf/pivotal-ui#260]
Currently neither of these examples work:

```
<SortableTable headers={[<TableHeader sortable={true}>Name</TableHeader>]}>
    <TableRow key={0}><TableCell>foo</TableCell></TableRow>
    <TableRow key={1}><TableCell>bar</TableCell></TableRow>
</SortableTable>

<SortableTable headers={[<TableHeader sortable={true}>First Name</TableHeader>,<TableHeader sortable={true}>Last Name</TableHeader>]}>
    <TableRow key={0}><TableCell>foo</TableCell><TableCell>bar</TableCell></TableRow>
</SortableTable>
```

Seems to be because we're expecting the `children` of the `SortableTable` and `TableRow` components to be arrays, but it seems like they shouldn't have to be. A table with one row in it is perfectly valid HTML.

**Comment by rdy** [comment]
This might be addressed in upgrading to react 0.14, in the future all react children will be plain arrays instead of objects. But if we want to fix this before this is released we might just want to switch to React.Children.map to fix it.

**Comment by atomanyih** [comment]
fixed in 43a8f0ae718d8c86fb6fed2fbd6bddaf3a823911

This should go out in 2.0.1

**Comment by atomanyih** [comment]
this is now available in 2.1.0-alpha.0 :tada:
